src/Widgets/ImageWidget/ImageWidgetClass.js

```diff
@@ -6,6 +6,18 @@ const ImageWidget = Scrivito.provideWidgetClass("ImageWidget", {
     alignment: ["enum", { values: ["left", "center", "right"] }],
     alternativeText: "string",
     link: "link",
+    animation: [
+      "enum",
+      {
+        values: [
```

Comment:
I would also add the value `"none"`.

Please make this also the default value:

```js
  initialContent: {
    animation: "none",
  },
```